taskcluster/scripts/toolchain/rustup-setup.sh

```diff
@@ -15,26 +15,19 @@ export SCCACHE_IDLE_TIMEOUT='1200'
 export SCCACHE_CACHE_SIZE='40G'
 export SCCACHE_ERROR_LOG='/builds/worker/sccache.log'
 export RUST_LOG='sccache=info'
-# We are pinned at this rust stable version (see also ../.circleci/config.yml)
-export RUST_STABLE_VERSION='1.50.0'
```

Comment:
Good news, we no longer need to keep this version in sync across multiple files, it's all done via `rust-toolchain` now.